controller/instance_handler.go

```diff
@@ -69,6 +69,20 @@ func (h *InstanceHandler) syncStatusWithInstanceManager(im *longhorn.InstanceMan
 		return
 	}
 
+	if im.Status.CurrentState == longhorn.InstanceManagerStateUnknown {
+		// Do not set the replica status to unknown state directly when the instance manager is
```

Comment:
I don't think we need this change. The essential of this issue is, the volume controller shouldn't mark unknown replicas as Failed.

Reply:
Yeah, you are right. Marking the replica as unknown and don't failed it is a good idea.
Updated the PR. Thank you.